### PR TITLE
feat(FEC-11666): Primary + Redirect token reports

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

fire `SOURCE_URL_SWITCHED` event with the original URL and the updated one

Depends on https://github.com/kaltura/playkit-js/pull/613

Solves FEC-11166

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/broadpeak.js [hunk 1]
```diff
@@ -171,8 +171,10 @@ class BroadPeak extends BasePlugin {
   _getSource(playbackUrl: string): void {
     this.session.getURL(playbackUrl).then(result => {
       if (!result.isError()) {
-        this.logger.debug('getUrl response', result.getURL());
-        this.sourceChangeCallback(result.getURL());
+        const updatedUrl = result.getURL();
+        this.logger.debug('getUrl response', updatedUrl);
+        this.sourceChangeCallback(updatedUrl);
+        this.dispatchEvent(this.player.Event.SOURCE_URL_SWITCHED, {origUrl: playbackUrl, updatedUrl});
```
Comment (OrenMe) [inline src/broadpeak.js:177]: lets use full names - originalUrl

File: src/broadpeak.js [hunk 2]
```diff
@@ -171,8 +171,10 @@ class BroadPeak extends BasePlugin {
   _getSource(playbackUrl: string): void {
     this.session.getURL(playbackUrl).then(result => {
       if (!result.isError()) {
-        this.logger.debug('getUrl response', result.getURL());
-        this.sourceChangeCallback(result.getURL());
+        const updatedUrl = result.getURL();
+        this.logger.debug('getUrl response', updatedUrl);
+        this.sourceChangeCallback(updatedUrl);
+        this.dispatchEvent(this.player.Event.SOURCE_URL_SWITCHED, {originalUrl: playbackUrl, updatedUrl});
```
Comment (RoyBregman) [inline src/broadpeak.js:177]: is this a valid JSON?

Comment (yairans) [reply src/broadpeak.js:177]: its not JSON. its js object

Comment (RoyBregman) [reply src/broadpeak.js:177]: same same :) still doesn't look valid :)

Comment (RoyBregman) [reply src/broadpeak.js:177]: But it is, so ok :)

**Review by RoyBregman — APPROVED** [review]

**Review by OrenMe — APPROVED** [review]

**Pull request merged**